.github/workflows/website-gh-pages.yml

```diff
@@ -29,26 +29,25 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Use Node.js node 18.x
-        uses: actions/setup-node@v3
-        with:
-          node-version: 18.x
-
-      - uses: pnpm/action-setup@v2
+      - uses: ./.github/actions/setup
 
       - name: Install dependencies
         run: pnpm install
 
+      - name: Install browsers
+        run: pnpm exec playwright install
+        working-directory: packages/playground-website
```

Comment:
Does this need to be `packages/typespec-azure-playground-website`?

Reply:
Good catch